google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/SelectConfigDestinationFolderDialog.java

```diff
@@ -63,19 +64,14 @@ public SelectConfigDestinationFolderDialog(
         FileChooserDescriptorFactory.createSingleFolderDescriptor()
     );
 
-    // Present a canonical target folder as default in the path field.
-    if (project != null && project.getBasePath() != null) {
-      if (fileType == ConfigFileType.APP_YAML) {
-        Path appYamlPath = Paths.get(project.getBasePath()
-            + AppEngineDeploymentRunConfigurationEditor.DEFAULT_APP_YAML_DIR);
-        destinationFolderChooser.setText(appYamlPath.toString());
-      } else if (fileType == ConfigFileType.DOCKERFILE) {
-        Path dockerfilePath = Paths.get(project.getBasePath()
-            + AppEngineDeploymentRunConfigurationEditor.DEFAULT_DOCKERFILE_DIR);
-        destinationFolderChooser.setText(dockerfilePath.toString());
-      } else {
-        destinationFolderChooser.setText(project.getBasePath());
+    // Suggests the parent of the specified file as directory.
```

Comment:
We're still suggesting a directory. We're feeding this method with the path from the facet settings, which is guaranteed to be there since the user doesn't have the option to (not) provide one, it's automatically done for them at framework adding time.